hawkbit-mgmt-api/src/main/java/org/eclipse/hawkbit/mgmt/json/model/rollout/AbstractMgmtRolloutConditionsEntity.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (c) 2015 Bosch Software Innovations GmbH and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.hawkbit.mgmt.json.model.rollout;
+
+import org.eclipse.hawkbit.mgmt.json.model.MgmtNamedEntity;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+/**
+ * Model for defining Conditions and Actions
+ */
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
+public abstract class AbstractMgmtRolloutConditionsEntity extends MgmtNamedEntity {
+
+    private MgmtRolloutCondition successCondition = new MgmtRolloutCondition();
+    private MgmtRolloutSuccessAction successAction = new MgmtRolloutSuccessAction();
```

Comment:
Do you really need these initializations? The empty constructor is only available because json creator.